.github/workflows/snapshot.yml

```diff
@@ -1,11 +1,9 @@
 name: Snapshot
 on:
-  push:
```

Comment:
Shouldn't the CLI still be published as a snapshot?  It's only the google deploy we want to avoid ... right?